backend/src/migrations/20240210124346_create_tables.ts

```diff
@@ -41,6 +22,28 @@ async function up(knex: Knex): Promise<void> {
             .notNullable()
             .defaultTo(knex.fn.now());
     });
+    await knex.schema.createTable(
```

Comment:
If files with previous migrations have already used on staging, you should consider to change DB structure by creating new migration file.